ingestion/functions/aws.py

```diff
@@ -12,8 +12,9 @@
 
 AWS_REGION = "us-east-1"
 AWS_IMAGE = {
-    'dev': "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:latest",
-    'prod': "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:stable"
+    "dev": "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:latest",
+    "qa": "612888738066.dkr.ecr.us-east-1.amazonaws.com/gdh-ingestor:latest",
```

Comment:
should this actually be on `stable`, so that qa is a prod-like environment? Or do we need an `rc` tag so that qa is what prod would become if we were happy with the behaviour in qa? To discuss, I think.

Reply:
+1 for qa -> stable